tests/algorithms/lstm_classifier/lstm_classifier_test.py

```diff
@@ -0,0 +1,66 @@
+import unittest
+import sys
+import os
+from unittest.mock import MagicMock
+
+sys.path.append(os.path.realpath("."))
+
+import src.algorithms.lstm_classifier.lstm_classifier as lstm_classifier
+
+
+class LstmClassifierTest(unittest.TestCase):
+    def test_get_articles_and_labels(self):
```

Comment:
Can you also check `original_label_to_clean_label` which is returned.

[returned]: https://github.com/intuit/fawkes/blob/8f4a0183b20efebc973ad6157da599439ad9dcc2/src/algorithms/lstm_classifier/lstm_classifier.py#L78